spec.bs

```diff
@@ -1493,6 +1511,28 @@ give to the [=Relying Party=] (e.g., the credit card number).
 NOTE: See [issue #77](https://github.com/w3c/secure-payment-confirmation/issues/77)
       for discussion on possible spec changes related to this section.
 
+## User opt out ## {#sctn-user-opt-out}
+
+The API option {{SecurePaymentConfirmationRequest/showOptOut}} provides the
+developer a way to ask the user agent to provide the user with an opportunity to
+indicate they wish to opt out of the relying party's storage of information.
```

Comment:
Editorial simplification (now that I reread it):

"The API option {{SecurePaymentConfirmationRequest/showOptOut}} tells the user agent to provide a way for the user to indicate they wish to opt out of the relying party's storage of information."

Reply:
Done